Restructure Source Maps Upload page (#5799)
getsentry/sentry-docs: src/platform-includes/sourcemaps/upload/webpack/javascript.mdx

Sentry uses [releases](/product/releases/) to match the correct source maps to your events.
You can use the Sentry Webpack plugin to automatically create releases and upload source maps to Sentry when bundling your app.

## Installation

```shell {tabTitle:npm}
npm install --save-dev @sentry/webpack-plugin
```

```shell {tabTitle:yarn}
yarn add --dev @sentry/webpack-plugin
```

## Configuration

> Learn more about the configuration of the plugin using our [Sentry Webpack Plugin documentation](https://github.com/getsentry/sentry-webpack-plugin).

Reviewer: ```suggestion
Learn more about configuring the plugin in our [Sentry Webpack Plugin documentation](https://github.com/getsentry/sentry-webpack-plugin).
```
